### ISSUE TITLE
[Bug/Feature]: Field to paste ETA template missing in Better BibTex Prefs for QuickCopy

### ISSUE THREAD

**Issue by acsr** [retorquere/zotero-better-bibtex#2526]
### Debug log ID

Q23ZJ4PJ-refs-euc

### What happened?

I am still missing aan issue similar to https://github.com/retorquere/zotero-better-bibtex/issues/2457  Where to paste ETA template? 

- The Better BibTeX prefernces open via Button in a seperate window.
     - The ``Export -> QuickCopy`` tab looks like the screeenshot
     - BUG: The former UI Input field is missing
     - Even after restarts!
- I am on Zotero 6.0.26 on an MBP M1 under MacOS Monterey, with:
    - Better BibTex for Zotero 6.7.79 autoinstalled at 09. Mai 2023
    - Calling the eta-template directly and via Zutilo 3.10.0 Shortcut

![Screenshot 2023-05-29 um 20 29 14](https://github.com/retorquere/zotero-better-bibtex/assets/400035/79f5e0a5-2bfa-46c1-ba2e-34f5c74db6a1)

- The eta-template code is reachable via Zotero Config-Editor in 
  ``extensions.zotero.translators.better-bibtex.quickCopyEta``
- The QuickCopy export using the template is executed well on this machine. 

- checked the commit in tag 6.7.63 fixing #2457 and compared it with current 6.7.79. 
    - https://github.com/retorquere/zotero-better-bibtex/commit/570c08df618abea5407bd7bfdb5e54501b04fa5e
    - and in content/Preferences/export/quickcopy.pug
        - https://github.com/retorquere/zotero-better-bibtex/commit/b6c9ee8cb546998a84ca0198253ec01383266564

I cannot find myself a location where the bug was reintroduced if still the same/similar to #2457 (no screenshot provided).

**Comment by acsr** [comment]
add proper title

**Comment by acsr** [comment]
Test build [test build 6.7.79.4359](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-6.7.79.4359.xpi) is not available for me (anymore?) even after login. didnt recognize the notification to give feedback, sorry

**Comment by retorquere** [comment]
> Test build [test build 6.7.79.4359](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-6.7.79.4359.xpi) is not available for me (anymore?)

Test builds expire after approximately one week.

**Comment by acsr** [comment]
It seems that the eta-template field is not available on next visit of the BetterBibTex Prefs as expected.

## Steps to reproduce
Close the BetterBibTex Prefs and reopen it:

- in the tab "Export" -> QuickCopy" the Input field for LaTeX citation shows up with label LaTeX command
- only after triggering a change to the dropdown e.g. to "Zotero select link" and then back to "Eta-template" unveils the field for the Eta-template.

My Zotero Prefs setting in tab Export  QuickCopy "Format for Entries" points to: "Better BibTeX Citation Key Quick Copy"

Screenshot of the initial view.
![image](https://github.com/retorquere/zotero-better-bibtex/assets/400035/be106cb5-b861-4ec7-a859-ee4d2ec36b76)

**Comment by acsr** [comment]
I can confirm that with [test build 6.7.79.4400](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-6.7.79.4400.xpi):

**The Eta-Template field now does not vanish and stays visible on initial visit of the tab. THX!**

The tight width of the textbox remains, as shown in my screenshot. 

But one thing after the other. 
Fixing two things at the same time is not a good idea.

I open a new ticket, and close this as resolved for me.
